tests/test_name_detail.py

```diff
@@ -21,9 +21,9 @@ def test_person_itemprops(client):
     response = client.get(
         reverse('name:detail', args=[name.name_id]))
 
-    assert 'itemprop=\"name\"' in response.content
-    assert 'itemprop=\'url\'' in response.content
-    assert 'itemprop=\"birthDate\"' in response.content
+    assert b'itemprop=\"name\"' in response.content
+    assert b'itemprop=\'url\'' in response.content
```

Comment:
This seems a little odd to me, is there a reason for the mixed quotes between this line and the next/previous?

Reply:
same comment for the other changes shown in this module

Reply:
It's the way it is designed in [name_detail.html](https://github.com/unt-libraries/django-name/blob/py2to3/name/templates/name/name_detail.html#L60) page.